presto-parser/src/test/java/com/facebook/presto/sql/parser/TestSqlParser.java

```diff
@@ -911,8 +934,39 @@ public void testSelectWithOrderBy()
                                         new Identifier("a"),
                                         ASCENDING,
                                         UNDEFINED)))),
+                                Optional.empty(),
                                 Optional.empty()),
                         Optional.empty(),
+                        Optional.empty(),
+                        Optional.empty()));
+    }
+
+    @Test
+    public void testSelectWithOffset()
+    {
+        assertStatement("SELECT * FROM table1 OFFSET 2 ROWS",
```

Comment:
Similar as elsewhere, could we verify a error when the OFFSET is specified incompletely?  Is it correct to allow a number > 1 that is specified with `ROW`?

Reply:
yes I add more UT checks on analyzer to catch and verify